__tests__/index.test.ts

```diff
@@ -44,6 +46,12 @@ const BLANK_NAME_3 = "test/blank3";
 const SECRET_FOR_BLANK_3 = '{"username": "integ", "password": "integpw", "config": {"id2": "example2"}}';
 const BLANK_ALIAS_INPUT_3 = "," + BLANK_NAME_3;
 
+
+const DEFAULT_TIMEOUT = '1000';
```

Comment:
The `DEFAULT_TIMEOUT` that is introduced in the CR is not used anywhere. Would you mind looking into that and cleaning it out. It seems that the mock is always using constant and the expectation is always having the number explicitly written out. Do whichever way you feel is better but it should be consistent.